Rakefile

```diff
@@ -1,20 +1,26 @@
-task :environment do
-  ENV["PLAYLISTER_ENV"] ||= "development"
-  require_relative 'config/environment'
-  # require 'logger'
-  # ActiveRecord::Base.logger = Logger.new(STDOUT)
-end
+require_relative 'config/environment.rb'
+require "sinatra/activerecord/rake"
 
 namespace :db do
-  task :migrate => :environment do
-    migrate_db
+
+  desc "Migrate the db"
```

Comment:
I believe you should get the `db:migrate` and `db:drop` tasks for free by requiring `'sinatra/activerecord/rake'` at the top - you should be able to remove these definitions.